src/mumble/Audio.cpp

```diff
@@ -25,13 +26,24 @@ LoopUser LoopUser::lpLoopy;
 CodecInit ciInit;
 
 void CodecInit::initialize() {
-	CELTCodec *codec = NULL;
+#ifdef USE_OPUS
+	OpusCodec *oCodec = new OpusCodec();
+	if (oCodec->isValid()) {
+		oCodec->report();
+		g.oCodec = oCodec;
+	} else {
+		qWarning("CodecInit: failed to load Opus, it will not be available for encoding/decoding");
```

Comment:
Should probably be encapsulated? `Failed to`

Reply:
I think the message could be more elaborative. "encoding/decoding audio". Maybe "load" could be improved. But it's okay.

Reply:
Done.